homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -258,6 +260,7 @@ class PlenticoreSensorEntityDescription(
         name="Autarky Day",
         native_unit_of_measurement=PERCENTAGE,
         icon="mdi:chart-donut",
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
From the key, it looks like this is a statistic value and not a measurement.
Is this value calculated? Is it a moving window of aggregation?

```suggestion
```

Reply:
It is computed on measurement and because of this

> For supported sensors, statistics of hourly min, max and average sensor readings is updated every 5 minutes.

I thought it would be nice to have the min and max values during the day. But I can also remove it again if you like.

Reply:
Please remove the state_class on all statistic sensors as in the developer docs it is specified the following:
> The state represents a measurement in present time, not a historical aggregation such as statistics or a prediction of the future. 

https://developers.home-assistant.io/docs/core/entity/sensor#available-state-classes

Reply:
Removed from all statistics sensors.